### ISSUE TITLE
Laravel 5.4 Compatibility

### ISSUE THREAD

**Issue by ecodrutz** [nWidart/laravel-menus#2]
Will this be upgraded to support Laravel 5.4?

**Comment by nWidart** [comment]
It will.

**Comment by noxify** [comment]
@nWidart - i have tested it with the laravelcollective/html 5.4 branch.
It's working fine - You only have to update the Service Provider, because the `share` method has been removed in 5.4.

https://laravel.com/docs/5.4/upgrade 

> The share method has been removed from the container. This was a legacy method that has not been documented in several years. If you are using this method, you should begin using the singleton method instead:
```
$container->singleton('foo', function () {
    return 'foo';
});
```

If you want, i can send you a PR with the changes :) 
Fork: https://github.com/noxify/laravel-menus

**Comment by nWidart** [comment]
That's good news 😄  
PR welcome indeed. Is there an ETA when laravelcollective will tag the release ?

**Comment by noxify** [comment]
Did'nt saw any date - maybe we can use a develop branch until they released it.

**Comment by noxify** [comment]
@nWidart / @ecodrutz  - can be closed now, or?

**Comment by nWidart** [comment]
Not yet. I've updated orchestra/testbench to `3.4`, but tests are crashing.

**Comment by nWidart** [comment]
```
PHP Fatal error:  Class 'PHPUnit\Framework\TestCase' not found in /Users/nicolaswidart/Sites/Packages/laravel-menus/vendor/orchestra/testbench/src/TestCase.php on line 22
```

**Comment by nWidart** [comment]
Hm when changing the following


```
use PHPUnit\Framework\TestCase as BaseTestCase;
```

To:

```
use PHPUnit_Framework_TestCase as BaseTestCase;
```

It works. Something on orchestra/testbench it seems

**Comment by nWidart** [comment]
Issue fixed and tagged as 0.3.0 👍
